src/components/row.js

```diff
@@ -93,9 +93,7 @@ function Row(props) {
         onMouseEnter={() => props.onHighlightState?.(state, props.index)}
         onMouseLeave={() => props.onHighlightState?.()}
         touchstart={() => props.onHighlightState?.(state, props.index)}
-        onClick={() => {
-          handleReveal();
-        }}
+        onClick={!props.total ? handleReveal : null}
```

Comment:
conditional event based on the input props

Reply:
In your solution, you are still attaching an event. I was saying don't attach event if it is 1total row`

Refer my solution:

You can do following on line no 77 
`const clickEvent = !props.total ? { 'onClick': handleReveal } : {};`

and then on line no 96 you can write  ` {... clickEvent}`
```
<tr className={props.total ? 'state is-total' : 'state'}
      onMouseEnter={() => props.onHighlightState?.(state, props.index)}
      onMouseLeave={() => props.onHighlightState?.()}
      touchstart={() => props.onHighlightState?.(state, props.index)}
      {... clickEvent}
>

```

Reply:
Also, resolve conflicts